homework/index.js

```diff
@@ -30,17 +34,110 @@
     return elem;
   }
 
+  function renderError(error) {
+    const body = document.body;
+    body.innerHTML = '';
```

Comment:
Clearing out the whole body is a bit drastic. Perhaps just clear our the root div? But see my other comments below what this root div should contain.